ci: pass $DISK_CONFIG properly to minikube start

Rakshith-R opened the pull request:
Having double quotes around $DISK_CONFIG led to these args
not being properly passed to minikube start. This commit fixes it.

Signed-off-by: Rakshith R <rar@redhat.com>

Rakshith-R committed:
ci: pass $DISK_CONFIG properly to minikube start

Having double quotes around $DISK_CONFIG led to these args
not being properly passed to minikube start. This commit fixes it.

Signed-off-by: Rakshith R <rar@redhat.com>

Madhu-1 reviewed (dismissed):
Please make sure you test your changes locally and working fine. because some of the changes won't get tested in CI.

Rakshith-R committed:
ci: use 0 as default NUM_DISKS in minikube.sh

This is done to prevent conflicts with current ci setup externally
attaching disks.

Signed-off-by: Rakshith R <rar@redhat.com>

Rakshith-R commented:
```
o further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.

Pool minikube created



Vol osd-0 created



Vol osd-1 created



Vol osd-2 created



Disk attached successfully



Disk attached successfully



error: Failed to attach disk

error: internal error: unable to execute QEMU command 'device_add': Duplicate ID 'virtio-disk3' for device



script returned exit code 1
```
https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.21/detail/mini-e2e-helm_k8s-1.21/1087/pipeline
Settign default `NUM_DISKS` to prevent conflicts.

Madhu-1 reviewed (approved):
May be send a followup PR to make it 1 or 3 as `0` is not helpful here.

Rakshith-R commented:
@mergifyio rebase

Rakshith-R commented:
/retest all

Rakshith-R commented:
> /retest all

`Error: Failed to download metadata for repo 'fedora-cisco-openh264': Cannot prepare internal mirrorlist: Status code: 503 for https://mirrors.fedoraproject.org/metalink?repo=fedora-cisco-openh264-34&arch=x86_64 (IP: 209.132.190.2)
`
